apps/src/aichat/views/AichatView.tsx

```diff
@@ -413,6 +414,7 @@ const AichatView: React.FunctionComponent<LabProps<AichatLevelProperties>> = ({
               headerContent={chatWorkspaceHeader}
               className={moduleStyles.panelContainer}
               headerClassName={moduleStyles.panelHeader}
+              rightHeaderContent={<AiChatHeaderButtons />}
```

Comment:
Maybe a later refactor for the AI Tutor/AI Chat team, but could be worth pulling in the `PanelContainer` into the shared Chat Workspace so the header doesn't have to be duplicated